src/client/settings/_settings.coffee

```diff
@@ -26,10 +26,17 @@ angular.module 'app.settings', []
   vm.load = ->
     $http.get('/containers/settings/settings').success (data) ->
       vm.settings = data
- 
+
+  vm.start_screens = ["Movies", "TV Series", "Anime"]
+
+  vm.movies_quality =
+    all: "All"
```

Comment:
aha
I can do that for consistency.The other two keys needed it because of how coffee-script parsed them